web/server/vue-cli/src/components/Product/ProductConfigForm.vue

```diff
@@ -39,6 +39,14 @@
       @input="productConfig.runLimit = $event || null"
     />
 
+    <v-select
```

Comment:
Extend the GUI userguide with this new field after this section and add a little description what is it for and what each item means: https://github.com/Ericsson/codechecker/blob/master/web/server/vue-cli/src/assets/userguide/userguide.md#disable-review-status-change